glusterfsd/src/glusterfsd-mgmt.c

```diff
@@ -325,6 +330,22 @@ glusterfs_handle_terminate(rpcsvc_request_t *req)
     }
     ctx = glusterfsd_ctx;
 
+    dict = dict_new();
+    if (xlator_req.dict.dict_len) {
+        ret = dict_unserialize(xlator_req.dict.dict_val,
+                               xlator_req.dict.dict_len, &dict);
+        if (ret < 0) {
+            gf_log(this->name, GF_LOG_ERROR,
+                   "Failed to unserialize "
+                   "req-buffer to dictionary");
+            goto err;
+        }
+    }
+
+    ret = dict_get_str(dict, "cluster.brick-graceful-cleanup", &value);
```

Comment:
It would be better to use a constant instead of explicit value for the name of the key.